### PR TITLE
Transaction Settings Smoke Test & some swap tests

### PR DESCRIPTION
**Pull request by MilanVojnovic95**
Written Smoke test for Transaction Settings, contains a check of all buttons and input fields on Transaction Setting and also checks the validation fields and checks if the hyprelinks buttons redirect to the correct address.

Added also tests for swapping tokens along with some subgraph facade which sends a requests to subgraph by cypress.

### CODE REVIEW

File: tests/cypress/integration/smoke/TransactionSettingsTests.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      })
+      it('Should not allow to type not numbers into Slippage Tolerance Field input [TC-50]', () => {
+        TransactionSettings.typeSlippageTolerance_Field('!#$%^&*(*)_qewruip')
+        TransactionSettings.enterValid_errorMesssage().should('be.visible')
+      })
+      it('Should not allow to type not numbers into Gas Price Field input [TC-50]', () => {
+        TransactionSettings.typeGasPrice_Field('!#$%^&*(*)_qewruip')
+        TransactionSettings.enterValid_errorMesssage().should('be.visible')
+      })
+    })
```
Comment (niemam29) [inline tests/cypress/integration/smoke/TransactionSettingsTests.ts:50]: Please use prettier formatter from the repo to keep consistent

**Review by niemam29 — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/smoke/TransactionSettingsTests.ts (hunk 1 above)
Comment (MilanVojnovic95) [reply tests/cypress/integration/smoke/TransactionSettingsTests.ts:50]: I have some problem with prettier on VS code which I am now trying to solve

Comment (MilanVojnovic95) [reply tests/cypress/integration/smoke/TransactionSettingsTests.ts]: Solved issue with prettier formatter and also fixed selectors naming convention for Transaction Settings, Rewards Page and Liquidity page.

File: tests/cypress/integration/smoke/TransactionSettingsTests.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { TransactionSettings } from '../../../pages/TransactionSettings'
+
+describe('Transactions Settings Smoke Test', () => {
+  before(() => {
+    cy.visit('http://localhost:3000/#/swap?chainId=1')
```
Comment (niemam29) [inline tests/cypress/integration/smoke/TransactionSettingsTests.ts:5]: We are not always sure on what url we are going to run tests and due to that i think we should use baseUrl value from cypress.json URL. Due to that the url from cy.visit can be simplified to "/#/swap?chainId=1" (rest of the url cypress gonna take from this baseUrl

**Review by niemam29 — CHANGES_REQUESTED** [review]

**Review by niemam29 — APPROVED** [review]
Lgtm!

**Review by denciu — APPROVED** [review]

**Pull request merged**